lib/streamlit/elements/media_proto.py

```diff
@@ -20,10 +20,87 @@
 from validators import url
 
 from streamlit import type_util
-from streamlit.proto import Video_pb2
+from streamlit.proto.Audio_pb2 import Audio as AudioProto
+from streamlit.proto.Video_pb2 import Video as VideoProto
```

Comment:
Mm... there isn't a perfect distinction between when we group elements into a single file and when we don't. On one extreme, we have "every element in its own file", which makes it really annoying for things like each alert level. On the other, we have every "every element in one file" aka the big bad DeltaGenerator. With this refactor, I'm trying to find a balance that makes sense.

So far I've been grouping together elements that share common code (like media here), and elements that use the same proto to pass to frontend (markdowns, alerts). If anything text_widgets is the one that makes the least sense, since those neither share code nor resuse protos...

I don't think this should be called _widgets, since (IIUC) neither st.audio nor st.video actually return a value, which is one of the defining characteristics of widgets in my head? But I could see a different argument being "widget is anything you can interact with", since audio and video have user-adjustable controls (but not e.g. st.image or st.markdown)...